### DISCUSSION TITLE
@brief Sending audio over ESPNow , peer channel information

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#686] [Q&A] [answered]
I am gong through example of esp now ,  i need to change peering channel  , my esp32  is connected with  wifi so the  peering channel may change after time to time , how can  I manage with your espnow audio example  ?

**Comment by pschatzmann** [comment]
Please feel free to extend the functionality!
If you have this working please share it with a pull request..

https://esp32.com/viewtopic.php?t=18291

**Reply by pschatzmann** [reply]
Did you try to set the channel in the config ?

**Comment by aqildad-create** [comment, marked as answer]
how can I set the channel ?? how the receiver know what the sender channel is ? 

we cant we fix the channel if we are using WIFI .. because router assign the channel . and it can change any time .

**Reply by pschatzmann** [reply]
Just read the link that I posted above. It shows how to assign the cannel!
